packages/aws-rfdk/lib/deadline/lib/spot-event-plugin-fleet.ts

```diff
@@ -463,7 +463,7 @@ export class SpotEventPluginFleet extends Construct implements ISpotEventPluginF
       renderQueue: props.renderQueue,
       workerSettings: {
         groups: props.deadlineGroups,
-        pools: props.deadlinePools,
+        pools: props.deadlinePools?.map(pool => pool.toLocaleLowerCase()),
```

Comment:
nit: This is technically not necessary since we are already converting to lowercase in [WorkerInstanceConfiguration](https://github.com/aws/aws-rfdk/blob/18a8098944f3599446c0f319eaf46b383724cfc3/packages/aws-rfdk/lib/deadline/lib/worker-configuration.ts#L324), so we could remove it.

Alternatively, we can keep this here, but we should use `this.deadlineGroups` in the above line so that we're passing lowercase versions for both pools and groups, and not half relying on `WorkerInstanceConfiguration` to lowercase them.

Reply:
We cannot relay on  WorkerInstanceConfiguration since it is running only when worker instance spawned.
There is another [PR](https://github.com/aws/aws-rfdk/pull/465) for creating Pools and groups so we have to have groups and pools names lowercased. 
Group names are converted to lowercase above but it is good catch that I used values from `props`. (Should be fixed)